torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -92,6 +92,11 @@ def can_auto_functionalize(op: torch._ops.OperatorBase) -> bool:
             and type(arg.type.getElementType()) is torch.TensorType
         ):
             continue
+        if (
+            type(arg.type) is torch.ListType
+            and type(arg.type.getElementType()) is torch.TensorType
+        ):
+            continue
```

Comment:
Can you change the comment below? It looks like we now support Tensor[]

Reply:
done